gallery/visualtests/nxButton.js

```diff
@@ -114,8 +115,20 @@ describe('NxButton', function() {
           focusAndHoverTest(selector));
     };
 
+    describe('Small NxButton section', function() {
```

Comment:
For consistency with the others
```suggestion
    describe('small styled buttons', function() {
```
Note when updating test names that you'll want to update the names of the images too, to avoid generating unnecessary new images and taking up even more space in git-lfs.

Reply:
> Note when updating test names that you'll want to update the names of the images too, to avoid generating unnecessary new images and taking up even more space in git-lfs.

I see there is such a block online 150 below.  Is that section redundant with this one?

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/77fe815c43acda9601e579b05302e602784189c6